microcosm_eventsource/stores/rollup.py

```diff
@@ -12,6 +12,24 @@
 from microcosm_eventsource.models.rollup import RollUp
 
 
+def with_labels(dct):
+    """
+    Add labels to all column clauses in the given dictionary of aggregate query clauses.
+
+    This is meant to be used with the output of `RollUpStore._aggregate` below, and serves
+    as a workaround for an issue in SQLAlchemy currently affecting 1.4.x.
+    We should remove this once the issue is fixed in the library.
```

Comment:
Should we add a GLOB ticket to this comment?

Reply:
fair enough - adding